### PR TITLE
Fix up some of the CI errors.

### PR DESCRIPTION
**Pull request by Sarvatt**
Please don't pull yet, testing the CI.

### CODE REVIEW

**Comment by Sarvatt** [comment]
Ready for review now.

**Comment by Sarvatt** [comment]
Both broken yamls were my own characters so it's ok to change them :)

File: profiles/base.yaml [hunk 1]
```diff
@@ -852,5 +852,4 @@ shard_thief_password:
 forge_container:
 spell_cycle:
 
-restock_shop:
-
+restock_shop: {}
```
Comment (rpherbig) [inline profiles/base.yaml]: ❓ This is already defined in `base-empty` as `{}`. Why is it defined that way here too? I expected it to remain `restock_shop:`...

**Comment by Sarvatt** [comment]
https://github.com/rpherbig/dr-scripts/blob/master/test/test_validate.rb#L22 only loads base.yaml and doesn't know about base-empty, I don't know how to fix that.

**Comment by rpherbig** [comment]
I suspect something like this would work (refactoring is left as an exercise to the reader)

```ruby
$test_settings = OpenStruct.new(YAML.load_file('profiles/base.yaml').merge(YAML.load_file('profiles/base-empty.yaml')).merge(settings))`
```

or maybe

```ruby
$test_settings = OpenStruct.new(YAML.load_file('profiles/base.yaml').merge(settings))`
YAML.load_file('profiles/base.yaml').empty_values.each { |name, value| $test_settings[name] ||= value }
```

**Comment by Sarvatt** [comment]
I just removed the test_validate.rb changes, figure I can submit at least part of the fixes.

**Pull request merged**